apps/monitoring/.prettierignore

```diff
@@ -0,0 +1,6 @@
+.next
+saleor/api.tsx
+pnpm-lock.yaml
+graphql/schema.graphql
+generated
+backend
```

Comment:
No need to ignore .auth-data.json or .env.local?

Reply:
there is root .gitignore